docs/_rtd/development.md

```diff
@@ -16,6 +17,28 @@ We recommend following the command line installer instructions as users have rep
  
 To test if `conda` has been added to your path, run `conda info` in your Terminal. If you get an error message, it means `conda` has not been added to your `PATH` environment variable yet. To fix, run `export PATH="/Users/yourname/anaconda3/bin:$PATH"`.
 
+**Apple Silicon Installation**
+
+You will need to install [*miniforge*](https://github.com/conda-forge/miniforge) first.
+Miniforge contains conda with native Apple Silicon support. There are a few installation options available, all generally work the same way. Consult the documentation if you wish to read about them (using Mamba vs Conda for example).
+
+1. Getting Miniforge
+   1. Install via homebrew (recommended):
+       ```sh
+       brew install miniforge
+       ```
+   2. Download and Install via the Unix-like platform commands
```

Comment:
Instead of a numbered list as sub-heading, I would title these option 1 and option 2 to make it clear that only one is required